llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -732,21 +800,44 @@ def tensor_hook(
                         monitor_process = None
 
                     # Spawn a new process to register the model.
-                    process = SpawnProcess(
-                        target=_register_model_with_run_id_multiprocess,
-                        kwargs={
-                            'mlflow_logger':
-                                mlflow_logger,
-                            'composer_logging_level':
-                                logging.getLogger('composer').level,
-                            'model_uri':
-                                local_save_path,
-                            'name':
-                                self.mlflow_registered_model_name,
-                            'await_creation_for':
-                                3600,
-                        },
-                    )
+                    # Slower method to register the model via log_model.
+                    if self.using_peft:
```

Comment:
i added back original logic to respect peft ptal @dakinggg

Reply:
seems right, will review thoroughly once CI passes